Sources/BuildSystemIntegration/BuildSystemManager.swift

```diff
@@ -327,10 +327,8 @@ package actor BuildSystemManager: QueueBasedMessageHandler {
 
   private var cachedTargetSources = RequestCache<BuildTargetSourcesRequest>()
 
-  /// The parameters with which `SourceFilesAndDirectories` can be cached in `cachedSourceFilesAndDirectories`.
   private struct SourceFilesAndDirectoriesKey: Hashable {
     let includeNonBuildableFiles: Bool
```

Comment:
`cachedSourceFilesAndDirectories` is going to have two almost-identical `SourceFilesAndDirectories`. Would it make sense to just include whether the file is buildable as part of `SourceFileInfo?

Reply:
Good idea 👍🏽